fragments/filter/README.md

```diff
@@ -0,0 +1,632 @@
+
+#### Example 4: AND cql-text (GET)
+
+```http
+/search?filter=sentinel:data_coverage > 50 AND eo:cloud_cover < 10 
+```
+
+#### Example 4: AND cql-json (POST)
+
+```json
+{
+    "filter": {
+        "and": [
+            {
+                "gt": [
+                    {
+                        "property": "sentinel:data_coverage"
+                    },
+                    50
+                ]
+            },
+            {
+                "lt": [
+                    {
+                        "property": "eo:cloud_cover"
+                    },
+                    10
+                ]
+            }
+        ]
+    }
+}
+```
+
+An 'or' is also supported, matching if either condition is true. Though it's not a sensible query you could get images that have full data 
+coverage or low cloud cover.
+
+### Example 5
+
+This uses the same queryables as Example 4.
+
+#### Example 5: OR cql-text (GET)
+
+```http
+/search?filter=sentinel:data_coverage > 50 OR eo:cloud_cover < 10 
+```
+
+#### Example 5: OR cql-json (POST)
+
+```json
+{
+  "filter": {
+    "or": [
+            {
+               "gt": [
+                  { "property": "sentinel:data_coverage" },
+                  50
+               ]
+            },
+            {
+               "lt": [
+                  { "property": "eo:cloud_cover" },
+                  10
+               }
+               ]
+           ]
+    }
+}
+```
+
+### Example 6: Temporal
+
+This uses the same queryables as Example 4.
+
+The only temporal operator required is `ANYINTERACTS`, which follows the same semantics as the existing 
+`datetime` parameter. This is effectively that the datetime or interval operands have any overlap between them.
+
+#### Example 6: ANYINTERACTS cql-text (GET)
+
+```http
+/search?filter=datetime ANYINTERACTS 2020-11-11
+```
+
+#### Example 6: ANYINTERACTS cql-json (POST)
+
+```json
+{
+  "filter": {
+      "anyinteracts": [
+        { "property": "datetime" },
+        "2020-11-11"
+      ]
+  }
+}
+```
+
+### Example 6: Spatial
+
+The only spatial operator that must be implemented is `INTERSECTS`. This has the same semantics as the one provided
+in the Item Search `intersects` parameter.  The `cql-text` format uses WKT geometries and the `cql-json` format uses 
+GeoJSON geometries.
+
+#### Example 6: INTERSECTS cql-text (GET)
+
+```http
+/search?filter=INTERSECTS(geometry,POLYGON((-77.0824 38.7886,-77.0189 38.7886,-77.0189 38.8351,-77.0824 38.8351,-77.0824 38.7886)))
+```
+
+#### Example 6: INTERSECTS cql-json (POST)
+
+```json
+{
+    "filter": {
+        "intersects": [
+                { "property": "geometry" } ,
+                {
+                   "type": "Polygon",
+                   "coordinates": [[
+                        [-77.0824, 38.7886], [-77.0189, 38.7886],
+                        [-77.0189, 38.8351], [-77.0824, 38.8351],
+                        [-77.0824, 38.7886]
+                    ]]
+                }
+        ]
+    },        
+}
+```
+
```

Comment:
Not required for getting the PR in, but it'd be nice to have some examples of the more advanced features, to show how powerful it can be. Maybe it's in a separate document, so we don't overwhelm people with a super long page of examples.

Reply:
https://github.com/radiantearth/stac-api-spec/issues/139